speechbrain/lobes/models/huggingface_whisper.py

```diff
@@ -65,13 +65,15 @@ def __init__(
         freeze=False,
         freeze_encoder=False,
         output_attentions=True,
+        output_hidden_states=False,
```

Comment:
I'd prefer to see "output_all_hiddens" instead of "output_hidden_states" as we already have something really similar to what you did with Wav2vec2 and we named it "output_all_hiddens" see: https://github.com/speechbrain/speechbrain/blob/develop/speechbrain/lobes/models/huggingface_wav2vec.py

Reply:
Thank you for your answer! I pushed the modifications. I changed the variable name, inserted it on the docstring, and also stack the tensors instead of returning a tuple... Now it is more similar to the wav2vec2 class.